#! /bin/sh -e
## gtimelog_path.dpatch
## Gustavo Franco <stratus@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Extra ui_file and icon_file checks to satisfy where the files
## are installed in Debian.

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch -f --no-backup-if-mismatch -p0 < $0;;
    -unpatch) patch -f --no-backup-if-mismatch -R -p0 < $0;;
    *)
        echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
        exit 1;;
esac

exit 0

--- gtimelog.py	2006-02-02 12:17:39.000000000 -0200
+++ gtimelog.py	2006-02-02 12:19:30.000000000 -0200
@@ -21,6 +21,10 @@
 ui_file = os.path.join(os.path.dirname(__file__), "gtimelog.glade")
 icon_file = os.path.join(os.path.dirname(__file__), "gtimelog-small.png")
 
+if not os.path.exists(ui_file):
+    ui_file = "/usr/share/gtimelog/gtimelog.glade"
+if not os.path.exists(icon_file):
+    icon_file = "/usr/share/pixmaps/gtimelog-small.png"
 
 def format_duration(duration):
     """Format a datetime.timedelta with minute precision."""
